### DISCUSSION TITLE
What is the best alternative for ItemsPresenter in Avalonia 11?

### DISCUSSION THREAD

**Discussion by SwiftExtender** [AvaloniaUI/Avalonia#17317] [General]
I`am porting lib from Avalonia  0.10 to Avalonia 11.0 and one of the breaking changes (https://github.com/AvaloniaUI/Avalonia/pull/9677) 

> ItemsPresenter now only generates containers for simple (non-virtualizing) panels

What the best alternative for this (ItemsPresenter ) now?

**Comment by maxkatz6** [comment]
ItemsControl
